### PR TITLE
[go/flaky] incorporate flaky test state

### PR DESCRIPTION
**Pull request by can-anyscale**
Incorporate flaky test state + render green metric which exclude flaky tests. Also use a little bit more clearer icon for flaky tests.

Screenshot 2024-01-19 at 8 42 32 AM

### CODE REVIEW

File: js/src/components/case.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ const TestCase: React.FC<Prop> = (props) => {
         >
           <Row>
             <Col>{props.case.name}</Col>
-            {props.case.is_labeled_flaky && <Col span={2}>❆</Col>}
+            {props.case.is_labeled_flaky && <Col span={2}> 🥶</Col>}
```
Comment (aslonnie) [inline js/src/components/case.tsx:50]: do you need the space before the icon?

what is this emoji? 🥶

**Review by aslonnie — APPROVED** [review]

File: js/src/components/case.tsx (hunk 1 above)
Comment (aslonnie) [reply js/src/components/case.tsx:50]: I think they tried to use the snowflake emoji

Comment (can-anyscale) [reply js/src/components/case.tsx:50]: i changed it to something more visible ;), the snowflake is so hard to see; the space helps with visibility too

Comment (can-anyscale) [reply js/src/components/case.tsx:50]: ah ok i'll follow up

**Pull request merged**